config/crd/kustomization.yaml

```diff
@@ -17,5 +17,13 @@ patchesStrategicMerge:
 #+kubebuilder:scaffold:crdkustomizecainjectionpatch
 
 # the following config is for teaching kustomize how to do kustomization for CRDs.
-configurations:
-- kustomizeconfig.yaml
+# configurations:
+# - kustomizeconfig.yaml
```

Comment:
Don't we still need to "teaching kustomize how to do kustomization for CRDs" ?

Reply:
Reverted the changes here.

(I doubt we still need this or not, so I was experimenting and saw no diff with or without it.)